Slider/Assets/Scripts/Map/Mountain/Minecart/Minecart.cs

```diff
@@ -459,12 +475,19 @@ private static Vector3 getDirectionAsVector(int dir)
             mcState = MinecartState.RepairParts;
```

Comment:
do UnityEvents not allow enums to be passed in for `UpdateState()`

Reply:
Nope, they do not!